### PR TITLE
Orgs using D: AdRoll

### PR DESCRIPTION
**Pull request by wilzbach**
> And I just got the go-ahead.

### CODE REVIEW

**Comment by CyberShadow** [comment]
I know this is silly, but can you rename `adroll.png` to something else so it bypasses ad blockers?

**Comment by wilzbach** [comment]
> to something else so it bypasses ad blockers?

Ideas? `roll_ad.png`?

**Comment by JackStouffer** [comment]
Best not to have ad in the name

**Comment by wilzbach** [comment]
> Best not to have ad in the name

Okay I renamed it to `da_roll.png`. Does that bypass your adblock @CyberShadow?

**Comment by CyberShadow** [comment]
Yep

**Comment by CyberShadow** [comment]
"Integral parts of Adroll's business on D. " Where's the verb?

File: orgs-using-d.dd [hunk 1]
```diff
@@ -51,15 +51,13 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING www.funkwerk.com/karriere/stellenangebote, hiring)
         )
     )
-    $(COMMENT
-    $(DORG Adroll, https://www.adroll.com, adroll.png,
-         Retargeting platform,
+    $(DORG AdRoll, https://www.adroll.com, da_roll.png,
+         Marketing Platform,
          $(HTTP tech.adroll.com/blog/data-science/2015/12/08/data-science_event_processing.html, Integral parts) of $(HTTP tech.adroll.com/blog/data-science/2015/08/25/factorization-machines.html,Adroll's business) on D.
```
Comment (appascoe) [inline orgs-using-d.dd]: Need a capital `R` here in `AdRoll's business`.

**Comment by appascoe** [comment]
Don't think it needs a verb, as it's just indicating links that describe parts of the business.

**Comment by wilzbach** [comment]
> Need a capital R here in AdRoll's business.

Oh damn - sorry!

> Don't think it needs a verb, as it's just indicating links that describe parts of the business.

I added "are built", does this make it clearer?

Disclaimer: I am not a native speaker:

**Pull request merged**